src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -242,7 +241,15 @@ private void CreateAgplAlert()
 
         private void RemoveAgplAlert()
         {
-            _serverStore.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.LicenseManager_AGPL3, null));
+            try
+            {
+                _serverStore.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.LicenseManager_AGPL3, null));
+            }
+            catch (Exception e)
+            {
+                // nothing we do can here, we'll try to remove it on next restart or when reloading the license
+                Logger.Info("Failed to remove the AGPL alert", e);
```

Comment:
What scenario did you see that caused this?

Also `if (Logger.IsOperationsEnabled)` and log that as `Operations`

Reply:
After starting the server and during license initialization (or when the license is renewed - every day), we tried to open a write tx in order to remove the AGPL notification.
The write was already taken by a cluster operation (global notifications storage is using the server store).

Reply:
Wouldn't we wait until this is completed?

Reply:
It throws `ThrowOnTimeoutWaitingForWriteTxLock` after 30 seconds.
The lock is held by `Follower thread from Remote connection` (we observed slow IO during that time).

Reply:
Pushed my changes. We'll skip deleting the notification when it doesn't exist (on a running it shouldn't exist anyway).